e2e/testcases/cli_test.go

```diff
@@ -1068,3 +1068,91 @@ func TestNomosVersion(t *testing.T) {
 		nt.T.Fatalf("Expected to find config-sync component in output:\n%s\n", string(out))
 	}
 }
+
+func TestNomosStatusNameFilter(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.NomosCLI)
+
+	nt.MustKubectl("apply", "-f", "../testdata/filter/bookinfo.yaml")
+	nt.MustKubectl("apply", "-f", "../testdata/filter/reposync-bookinfo.yaml")
+	nt.MustKubectl("apply", "-f", "../testdata/filter/rootsync-crontab.yaml")
+	nt.MustKubectl("apply", "-f", "../testdata/filter/reposync-crontab.yaml")
```

Comment:
These will left the resources around after the test finishes, which causes cleanup issues in the shared test environment.

Please use the test framework to set up multiple R*Syncs.

Reply:
test setup has been changed to use the builtin from framework. All config files removed too